vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -316,29 +328,50 @@ and
                 projectContext.RemoveSourceFile(file)
                 updated <- true
         
-        let updatedRefs = site.AssemblyReferences() |> hashSetIgnoreCase
+        (*
```

Comment:
That's fine with this PR - the sources/flags notifier (which eventually calls through to this) is running on the UI thread.